designer/server/src/test/resources/designer-api-test.conf

```diff
@@ -0,0 +1,191 @@
+db {
+  url: "jdbc:hsqldb:mem:testDB;sql.syntax_ora=true"
+  driver: "org.hsqldb.jdbc.JDBCDriver"
+  user: "SA"
+  password: ""
+  connectionTimeout: 30000
+}
+
+authentication:  {
+  method: "BasicAuth"
+  usersFile: "designer/server/src/test/resources/basicauth-users.conf"
+}
+
+http {
```

Comment:
Why we need another config? Can't we use just designer.conf? Variety of config make changes in config structure hard

Reply:
I'd not say "hard". Rather "laborious". However, having multiple conf files prepared to be used in some contexts is more readable. 

But ok, I'm not aware yet of all specs contexts in Nu, so I refactor it to use the existing designer.conf (I'm changing part of it in the `WithMockableDeploymentManager` trait.